script/develop

```diff
@@ -6,17 +6,24 @@ set -e
 
 cd "$(dirname "$0")/.."
 
-mkdir -p ./site/public
-
-curl -sSLf https://dev--home-assistant-analytics.netlify.app/data.json | jq . > ./site/public/data.json
-curl -sSLf https://dev--home-assistant-analytics.netlify.app/custom_integrations.json | jq . > ./site/public/custom_integrations.json
-curl -sSLf https://dev--home-assistant-analytics.netlify.app/addons.json | jq . > ./site/public/addons.json
-curl -sSLf https://rc.home-assistant.io/integrations.json | jq . > ./site/public/integration_details.json
+rm -rf ./site/dist
+mkdir -p ./site/src/_data
 
+if [ "$ANALYTICS_DATA" = "prod" ]; then
```

Comment:
Shouldn't we just make this the default? We can assume most people will want to work on the UI, not new data storage